mailpoet/assets/js/src/newsletters/sending_status.jsx

```diff
@@ -150,8 +150,8 @@ function ListingItem({
     'has-row-actions'
   );
   let status = MailPoet.I18n.t('unprocessed');
-  if (processed === '1') {
-    if (failed === '1') {
+  if (processed) {
```

Comment:
I'm still setting a few errors in my browser console that are related to the change of the data type of some fields from string to integer. Maybe there are still other places that the jsx code needs to be updated in a similar way that you did here?

Below are the errors that I'm seeing. To you see those as well?

```
Warning: Failed prop type: Invalid prop `failed` of type `number` supplied to `ListingItem`, expected `string`.
ListingItem@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:160:7
ListingItem@webpack-internal:///./assets/js/src/listing/listing_item.jsx:26:5
tbody
ListingItems@webpack-internal:///./assets/js/src/listing/listing_items.jsx:24:1
table
div
Listing@webpack-internal:///./assets/js/src/listing/listing.jsx:48:5
C@webpack-internal:///./node_modules/react-router/esm/react-router.js:706:31
SendingStatusListing<@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:93:7
SendingStatus@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:41:5
Route@webpack-internal:///./node_modules/react-router/esm/react-router.js:449:29
Switch@webpack-internal:///./node_modules/react-router/esm/react-router.js:651:29
Router@webpack-internal:///./node_modules/react-router/esm/react-router.js:80:30
HashRouter@webpack-internal:///./node_modules/react-router-dom/esm/react-router-dom.js:103:35
App@webpack-internal:///./assets/js/src/newsletters/newsletters.jsx:237:85
```

```
Warning: Failed prop type: Invalid prop `taskId` of type `number` supplied to `ListingItem`, expected `string`.
ListingItem@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:160:7
ListingItem@webpack-internal:///./assets/js/src/listing/listing_item.jsx:26:5
tbody
ListingItems@webpack-internal:///./assets/js/src/listing/listing_items.jsx:24:1
table
div
Listing@webpack-internal:///./assets/js/src/listing/listing.jsx:48:5
C@webpack-internal:///./node_modules/react-router/esm/react-router.js:706:31
SendingStatusListing<@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:93:7
SendingStatus@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:41:5
Route@webpack-internal:///./node_modules/react-router/esm/react-router.js:449:29
Switch@webpack-internal:///./node_modules/react-router/esm/react-router.js:651:29
Router@webpack-internal:///./node_modules/react-router/esm/react-router.js:80:30
HashRouter@webpack-internal:///./node_modules/react-router-dom/esm/react-router-dom.js:103:35
App@webpack-internal:///./assets/js/src/newsletters/newsletters.jsx:237:85
```

```
Warning: Failed prop type: Invalid prop `processed` of type `number` supplied to `ListingItem`, expected `string`.
ListingItem@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:160:7
ListingItem@webpack-internal:///./assets/js/src/listing/listing_item.jsx:26:5
tbody
ListingItems@webpack-internal:///./assets/js/src/listing/listing_items.jsx:24:1
table
div
Listing@webpack-internal:///./assets/js/src/listing/listing.jsx:48:5
C@webpack-internal:///./node_modules/react-router/esm/react-router.js:706:31
SendingStatusListing<@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:93:7
SendingStatus@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:41:5
Route@webpack-internal:///./node_modules/react-router/esm/react-router.js:449:29
Switch@webpack-internal:///./node_modules/react-router/esm/react-router.js:651:29
Router@webpack-internal:///./node_modules/react-router/esm/react-router.js:80:30
HashRouter@webpack-internal:///./node_modules/react-router-dom/esm/react-router-dom.js:103:35
App@webpack-internal:///./assets/js/src/newsletters/newsletters.jsx:237:85
```

```
Warning: Failed prop type: Invalid prop `subscriberId` of type `number` supplied to `ListingItem`, expected `string`.
ListingItem@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:160:7
ListingItem@webpack-internal:///./assets/js/src/listing/listing_item.jsx:26:5
tbody
ListingItems@webpack-internal:///./assets/js/src/listing/listing_items.jsx:24:1
table
div
Listing@webpack-internal:///./assets/js/src/listing/listing.jsx:48:5
C@webpack-internal:///./node_modules/react-router/esm/react-router.js:706:31
SendingStatusListing<@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:93:7
SendingStatus@webpack-internal:///./assets/js/src/newsletters/sending_status.jsx:41:5
Route@webpack-internal:///./node_modules/react-router/esm/react-router.js:449:29
Switch@webpack-internal:///./node_modules/react-router/esm/react-router.js:651:29
Router@webpack-internal:///./node_modules/react-router/esm/react-router.js:80:30
HashRouter@webpack-internal:///./node_modules/react-router-dom/esm/react-router-dom.js:103:35
App@webpack-internal:///./assets/js/src/newsletters/newsletters.jsx:237:85
```